includes/wikia/api/DesignSystemApiController.class.php

```diff
@@ -195,4 +203,48 @@ private function isWikiaOrgCommunity() {
 
 		return wfGetBaseDomainForHost( parse_url( $wgServer, PHP_URL_HOST ) ) === $wgWikiaOrgBaseDomain;
 	}
+
+
+	/**
+	 * External API request to fandom store
+	 */
+	public function getFandomStoreData() {
+		global $wgCityId, $wgMemc, $wgFandomStoreMap;
+
+		// api for fandom store
+		$uri = 'http://138.201.119.29:9420/ix/api/seo/v1/footer'; // https://shop.fandom.com/ix/api/seo/v1/footer 500 - Internal Server Error
```

Comment:
Please consider moving that to config and removing it from git history.

Reply:
I'm not quite sure what `config` you're referring to? You used that term with the `fandomStoreMap` referring to **VariablesBase.php**. Doesn't this get saved to git as well?

Reply:
@cschucode I should be more specific. I'd put that in `config` repo in `secrets.php`

Reply:
I created `wgFandomShopUrl` and storing in **config**. Here's the pr for that https://github.com/Wikia/config/pull/2573